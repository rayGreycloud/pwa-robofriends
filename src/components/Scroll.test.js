import React from 'react';
import { shallow } from 'enzyme';

import Scroll from './Scroll';

describe('<Scroll />', () => {
  it('should render component', () => {
    const mockChildren = <div>Child</div>;
    expect(shallow(<Scroll children={mockChildren} />)).toMatchSnapshot();
  });
});
