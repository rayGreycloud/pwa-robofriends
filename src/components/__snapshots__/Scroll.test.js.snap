// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Scroll /> should render component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Scroll>
    <div>
        Child
    </div>
</Scroll>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div>
        Child
</div>,
      "style": Object {
        "border": "5px solid black",
        "height": "800px",
        "overflow": "scroll",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Child",
      },
      "ref": null,
      "rendered": "Child",
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div>
          Child
</div>,
        "style": Object {
          "border": "5px solid black",
          "height": "800px",
          "overflow": "scroll",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Child",
        },
        "ref": null,
        "rendered": "Child",
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
